src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -76,7 +76,7 @@ def load_pretrained_quantization(model: Module, model_name_or_path: str):
     model_path = get_safetensors_folder(model_name_or_path)
     state_dict = get_quantization_state_dict(model_path)
 
-    for name, submodule in iter_named_leaf_modules(model):
+    for name, submodule in iter_named_modules(model):
```

Comment:
I think this name is misleading since we aren't actually getting all the layers. Could we clarify we are targetting specifically modules that are possible to quantize? I also think we should maintain the original leaf helper